src/extension/wps/src/main/java/au/org/emii/gogoduck/worker/GoGoDuckModule.java

```diff
@@ -37,104 +46,126 @@ public URIList getUriList() throws GoGoDuckException {
     }
 
     public void postProcess(File file) {
-        return;
+        try {
+            Object layerKey = GoGoDuckConfig.getPropertyKeyByValuePart(profile);
+            if (layerKey != null) {
+                String layer = (String) layerKey;
+                String postProcessProperty = String.format("%s.postprocess", layer);
+                if (GoGoDuckConfig.properties.containsKey(postProcessProperty) && GoGoDuckConfig.properties.getProperty(postProcessProperty).equals("true")) {
+                    Method method = this.getClass().getDeclaredMethod(String.format("postProcess_%s", layer), File.class);
+                    method.invoke(this, file);
+                }
+            }
+        } catch (Exception e) {
+            throw new GoGoDuckException(String.format("Could not post process file '%s'", file.toPath()));
+        }
     }
 
-    public List<String> ncksExtraParameters() {
-        List<String> ncksExtraParameters = new ArrayList<String>();
-        return ncksExtraParameters;
+    public void postProcess_srs_oc(File file) {
```

Comment:
Can identify whether a variable is packed by checking ucar.nc2.dataset.VariableDS.hasScaleOffset().  When true the dataset has been packed.